e2e/vaultsecrets/input/non-default_wi.nomad.hcl

```diff
@@ -26,7 +26,7 @@ job "secrets" {
       }
 
       vault {
-        policies = ["access-secrets-TESTID"]
+        role = "TESTID"
```

Comment:
Not for this PR, but at some point we should find all the places we're using that `ReplaceInJobSpec` method and swap them out for HCL2 variables.